bedrock/mozorg/templates/mozorg/about/governance/policies/reporting.html

```diff
@@ -57,7 +57,7 @@ <h2>{{ _('How to Give a Report') }}</h2>
   </p>
 
   <p>
-  {% trans community_hotline='https://events.mozilla.org/communityparticipationhotline' %}
```

Comment:
Not related to this change, but the blockquote on this page feels a little overpowering. Worth adding a custom style to make it a bit less in-your-face?

![image](https://user-images.githubusercontent.com/400117/72529841-fdda0e80-3865-11ea-824a-d9cd7c97f65e.png)

Reply:
Yeah, it's a bit much, especially on such a lengthy chunk of text. We should revisit this in Protocol but for now I've added a custom style just for this page.